site/src/content/docs/contribute/style-guide.mdx

```diff
@@ -4,7 +4,7 @@ title: Documentation Style Guide
 
 ### Introduction
 
-Welcome to the Defense Unicorns Project Documentation Style Guide, your guide to writing style and terminology for all project communication. This style guide provides editorial guidelines for writing clear and consistent Unicorn-related documentation. Writing is an essential aspect of open-source software development, including documentation, user guides, and communication with the community. Consistency and clarity are critical in conveying our information effectively.
+Welcome to the Defense Unicorns Project Documentation Style Guide, your guide to writing style and terminology for all project communication. This style guide provides editorial guidelines for writing clear and consistent Unicorn-related documentation. Writing is an essential aspect of open source software development, including documentation, user guides, and communication with the community. Consistency and clarity are critical in conveying our information effectively.
```

Comment:
Nit - feel free to say out of scope - should we be actively looking to remove unicorn-related language?